bin/travis-ci/docker_publish.sh

```diff
@@ -20,5 +20,11 @@ set -eo pipefail
 # Log into our Docker registry
 echo "${DOCKER_REGISTRY_PASSWORD}" | docker login -u "${DOCKER_REGISTRY_USER}" --password-stdin "${DOCKER_REGISTRY_URL}"
 
-# Push image
-docker-compose --file docker-compose.yml --file dc.build.yml push nginx-php-moodle
+repository="$(docker images | grep nginx-php-moodle | awk '{print $1}')"
```

Comment:
I think this is the full image URL, not just the repository?

Reply:
The column in the output is `repository` so I kept it consistent

```bash
REPOSITORY                                          TAG                 IMAGE ID            CREATED             SIZE
docker.sdelements.com:443/moodle/nginx-php-moodle   v0.1.12             d09b795a87b7        11 days ago         475MB
```

Reply:
Hmm it's not really clear what the value is if you just read the script though.

Reply:
https://docs.docker.com/glossary/?term=repository - it aligns with Docker's definition

Reply:
alright `repository_url` or `repository_path` would be more clear to me but doesn't matter.